### PR TITLE
update README

### PR DESCRIPTION
**Pull request by dox2hc**
please approve

### CODE REVIEW

**Comment by dox2hc** [comment]
hello

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # selfpractice
-this repo is for hackerRank practicing.
```
Comment (DTrXuan) [inline README.md:2]: test review comment

**Comment by DTrXuan** [comment]
Hello

**Review by DTrXuan — APPROVED** [review]
change is OK!

**Pull request merged**